dandiapi/api/services/metadata/__init__.py

```diff
@@ -82,8 +85,6 @@ def version_aggregate_assets_summary(version: Version):
     Version.objects.filter(id=version.id, version='draft').update(
         modified=timezone.now(), metadata=version.metadata
     )
-    version.refresh_from_db()
-    return version
```

Comment:
Is the idea here that there's no value in doing a refresh from db because the version could become stale again immediately after? Because the call to `refresh_to_db` _would_ update the `version` instance with the new changes from the prior update query, right?

Reply:
> Is the idea here that there's no value in doing a refresh from db because the version could become stale again immediately after?

Yes. I think the goal long-term should be to use `refresh_from_db` only when staleness isn't a concern: testing, returning from an endpoint, etc. Plus, this function doesn't have any callers that store the return value, so I removed it entirely :P.